src/igvfd/schemas/variant.json

```diff
@@ -0,0 +1,57 @@
+{
+    "title": "Variant",
+    "description": "Abstract schema for variants.",
+    "$id": "/profiles/variant.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "selection_criteria": {
+            "title": "Selection Criteria",
+            "type": "array",
+            "items": {
+                "title": "The reason this variant was selected for study.",
+                "type": "string"
+            }
+        },
+        "ref": {
+            "title": "Reference Sequence",
+            "description": "The sequence found in this position in the reference.",
+            "type": "string",
+            "pattern": "^[ATCG]*$"
+        },
+        "alt": {
+            "title": "Variant Sequence",
+            "description": "The sequence found in this position in the variant.",
+            "type": "string",
+            "pattern": "^[ATCG]*$|^-$"
+        },
+        "assembly": {
+            "title": "Mapping Assembly",
+            "description": "The genome assembly to which coordinates relate. e.g. GRCh38.",
```

Comment:
title "Genome Assembly"

description "The genome assembly to which variant coordinates relate."